packages/monaco/src/browser/monaco-outline-contribution.ts

```diff
@@ -64,6 +68,38 @@ export class MonacoOutlineContribution implements FrontendApplicationContributio
                 });
             }
         });
+
+        this.editorManager.onCurrentEditorChanged(widget => {
+            if (widget) {
+                this.addConfigTabbingWidget();
+
+                widget.editor.onEditorOptionsChanged(() => {
+                    this.removeConfigTabbingWidget();
```

Comment:
Would it be possible to update the widget with the appropriate editor instead of removing/adding the widget? I might be wrong and it may be too complicated for this use case, but I was just wondering.

Reply:
You are right. After looking into the statusBar.add / remove function, I found the statusBar.entries is a Map, which means, I don't need to call removeConfigTabbingWidget() at all, as calling addConfigTabbingWidget() would always reset the entry that has the same entryId.